.eslintrc.js

```diff
@@ -199,7 +199,9 @@ module.exports = {
     "import/no-extraneous-dependencies": [
       "error",
       {
-        devDependencies: false,
+        // TODO This one is to make it work with unit tests.
+        // Perhaps a better solution could be a different config for both?
+        devDependencies: true,
```

Comment:
we can use a glob pattern to apply this rule to true only for tests files:

`"import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/*.test.js", "**/*.spec.js"]}]`

https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md

Reply:
:O good catch! That's the better solution

Reply:
done